nengo_viz/static/viz_ace.css

```diff
@@ -0,0 +1,6 @@
+#editor {
+    position: fixed;
```

Comment:
I'm a bit surprised about the amount of elements with `position: fixed` in the code base. I had to change quite a number of those to `absolute` (or sometimes to `relative`). I wouldn't be surprised if this were another case where it should be `absolute`.